common/types/types.ts

```diff
@@ -6,7 +8,12 @@ export type Word = {
   images: string[];
 };
 
-export type LanguageCode = string;
+export type LanguageCode = Opaque<string, "LanguageCode">;
```

Comment:
Should we perhaps move this into its own file, and the `makeLanguageCode` into a `util` file?